src/Navigation/Drawer/DrawerStyle.ts

```diff
@@ -64,15 +84,34 @@ export const DrawerWrapper = styled.div<DrawerWrapperProps>`
   width: 80vw;
   max-width: 300px;
   top: 0px;
-  right: 0px;
+  ${({ position }) =>
+    ({
+      left: `
+    left: 0px;
+  `,
+      right: `
+    right: 0px;
+  `,
+    }[position])}
+
   background-color: white;
-  animation: ${({ open }) => (open ? sideToLeft : sideToRight)} 0.3s ease-in-out;
-  transform: ${({ open }) => (open ? 'translateX(0%)' : 'translateX(100%)')};
+  animation: ${({ open, position }) =>
+      ({
+        right: open ? openFromRight : closeToRight,
+        left: open ? openFromLeft : closeToLeft,
+      }[position])}
+    0.3s ease-in-out;
+  transform: ${({ open, position }) =>
+    ({
+      right: open ? 'translateX(0%)' : 'translateX(100%)',
+      left: open ? 'translateX(0%)' : 'translateX(-100%)',
+    }[position])};
   outline: none;
   box-shadow: -2px 0 8px rgba(0, 0, 0, 0.15);
   overflow-y: auto;
 `;
 
 interface DrawerWrapperProps {
   open: boolean;
+  position: 'left' | 'right';
```

Comment:
Would be better to create a type for this in the component, and use it here. 
In case of changes it would be easier to update just one type, instead of updating in the style and the component files.

Reply:
Done!